tests/components/dlna_dmr/test_init.py

```diff
@@ -0,0 +1,66 @@
+"""Tests for the DLNA DMR __init__ module."""
+from unittest.mock import Mock, patch
+
+from homeassistant.components.dlna_dmr.config_flow import DlnaDmrFlowHandler
+from homeassistant.components.dlna_dmr.const import DOMAIN as DLNA_DOMAIN
+from homeassistant.components.media_player.const import DOMAIN as MEDIA_PLAYER_DOMAIN
+from homeassistant.const import CONF_NAME, CONF_PLATFORM, CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.setup import async_setup_component
+
+from .conftest import GOOD_DEVICE_LOCATION
+
+from tests.common import MockConfigEntry
+
+
+async def test_import_flow_started(
+    hass: HomeAssistant, mock_ssdp_scanner: Mock
+) -> None:
+    """Test import flow of YAML config is started if there's config data."""
+    mock_import_data = {
+        CONF_PLATFORM: DLNA_DOMAIN,
+        CONF_URL: GOOD_DEVICE_LOCATION,
+    }
+
+    mock_config: ConfigType = {
+        MEDIA_PLAYER_DOMAIN: [
+            {
+                CONF_PLATFORM: DLNA_DOMAIN,
+                CONF_URL: GOOD_DEVICE_LOCATION,
+            },
+            {
+                CONF_PLATFORM: "other_domain",
+                CONF_URL: GOOD_DEVICE_LOCATION,
+                CONF_NAME: "another device",
+            },
+        ]
+    }
+
+    with patch.object(
```

Comment:
https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations